tests/h/cli/commands/normalize_uris_test.py

```diff
@@ -131,33 +131,27 @@ def test_it_deletes_duplicate_document_meta_objects(req):
 
 
 @pytest.mark.usefixtures('index')
-def test_it_normalizes_annotation_target_uri(req):
-    annotation_1 = models.Annotation(userid='luke',
-                                     _target_uri='http://example.org/',
-                                     _target_uri_normalized='http://example.org')
-    annotation_2 = models.Annotation(userid='luke',
-                                     _target_uri='http://example.net/',
-                                     _target_uri_normalized='http://example.net')
-
-    req.db.add_all([annotation_1, annotation_2])
-    req.db.flush()
+def test_it_normalizes_annotation_target_uri(req, factories, db_session):
+    annotation_1 = factories.Annotation(userid='luke', target_uri='http://example.org/')
+    annotation_1._target_uri_normalized = 'http://example.org'
+    annotation_2 = factories.Annotation(userid='luke', target_uri='http://example.net/')
+    annotation_2._target_uri_normalized = 'http://example.net'
+    db_session.flush()
```

Comment:
Sure. But this flush here is so that the `_target_uri_normalized` values are getting flushed.
This is the only (and ugly) way of making sure the database fixtures have old normalized URIs, unfortunately.